meson.build

```diff
@@ -0,0 +1,35 @@
+project('bongocat-gnu',
+  'cpp',
+  license: 'GPL-3.0',
+  version: '0.1.0')
+
+sources = files([
+  'src/ctb.cpp',
+  'src/custom.cpp',
+  'src/data.cpp',
+  'src/input.cpp',
+  'src/jsoncpp.cpp',
+  'src/main.cpp',
+  'src/mania.cpp',
+  'src/osu.cpp',
+  'src/system.cpp',
+  'src/taiko.cpp',
+])
+
+cpp_flags = ['-std=c++17']
```

Comment:
You probably want to set this via the project options:

```meson
project(
    ...,
    default_options: ['cpp_std=c++17'],
)
```

Reply:
Thanks for the review, I will take that into account next time I'll be editing meson files.